Lib/test/test_monitoring.py

```diff
@@ -1811,14 +1812,13 @@ def test_gh108976(self):
 class TestOptimizer(MonitoringTestBase, unittest.TestCase):
 
     def setUp(self):
-        import _testinternalcapi
+        _testinternalcapi = import_module("_testinternalcapi")
         self.old_opt = _testinternalcapi.get_optimizer()
         opt = _testinternalcapi.new_counter_optimizer()
         _testinternalcapi.set_optimizer(opt)
         super(TestOptimizer, self).setUp()
 
     def tearDown(self):
-        import _testinternalcapi
```

Comment:
But this is still needed.

Reply:
> But this is still needed.

`tearDown` will not run if we skip in `setUp`, no?

Reply:
From where it will get `_testinternalcapi` if it run?

Reply:
Put it back with bdd8cff169999a49cd3feae20b73b5dd06e28687 😄